build.gradle

```diff
@@ -633,6 +633,7 @@ task allJavadoc(type: Javadoc, group: 'Documentation') {
             project(':checker-qual').sourceSets.main.allJava,
             project(':checker').sourceSets.main.allJava,
             project(':framework').sourceSets.main.allJava,
+            project(':framework').sourceSets.test.allJava,
```

Comment:
This is great and reveals 14 errors in javadoc!
Should the same be done for `:checker`?

Reply:
Sounds good to me! I will udpate the gradle file.

Reply:
There is also `dataflow/src/test`, `checker/src/testannotations`, and `checker-util/src/test`.
Please also fix the javadoc errors revealed by the new check.

Reply:
Okay, sounds good.

Reply:
Maybe framework-test/src/test as well?

Reply:
I will leave do `checker/src/testannotations` later in a new PR. Looks like we need to define the source set for testannotations folder.